tests/core/test_scripts.py

```diff
@@ -3092,6 +3098,68 @@ def test_do_run(self):
         assert 1 == customlist.size
 
 
+class TestCustomListUpdateEntriesScript(EndToEndSearchTest):
+    def populate_works(self):
+        self.populated_books: list[Work] = [
+            self._work(with_license_pool=True, title="Populated Book") for _ in range(5)
+        ]
+        self.unpopular_books: list[Work] = [
```

Comment:
Is this on purpose that one is popu**lated** and other unpopu**lar**? Should they be (`populated` and `unpopulated`) or (`popular` and `unpopular`) or this is on purpose? 😄

Reply:
Yes, I was bored